src/main/java/htsjdk/io/HtsPath.java

```diff
@@ -282,19 +283,36 @@ private URI getURIForString(final String pathString) {
     }
 
     /**
-     * check that there isn't a non file scheme at the start of the path
-     * @param pathString
-     * @param cause
+     * Check for problems associated with the presence of a heirachical scheme.
```

Comment:
hierarchical

Reply:
hieirearchicahcl

Reply:
I wish intellij was more consistent in its spell checking, it just ignores some stuff and I don't know why